### PR TITLE
Feat/triggerblobdata

### PR DESCRIPTION
**Pull request by samartse**
@BenMoon

### CODE REVIEW

File: pymepixviewer/main.py [hunk 1]
```diff
@@ -135,7 +136,15 @@ def __init__(self, timepix_ip, parent=None):
 
     def switchToMode(self):
         self._timepix.stop()
-        if self._current_mode is ViewerMode.TOA:
+        if True: #self._current_mode is ViewerMode.ViewerMode.Trig:
```
Comment (BenMoon) [inline pymepixviewer/main.py]: Maybe this `if` should be removed as it is always `true` or actually make this the `else`.

File: pymepixviewer/main.py [hunk 2]
```diff
@@ -415,11 +424,19 @@ def onModeChange(self, value):
 
     def onData(self, data_type, event):
 
+        print(data_type)
```
Comment (BenMoon) [inline pymepixviewer/main.py]: Do we want these prints in there? If we need an output here, this should rather be `logging`.

File: pymepixviewer/panels/blobview.py [hunk 3]
```diff
@@ -299,6 +299,8 @@ def onCentroid(self, event):
             self.updateBlobData(cluster_shot, cluster_x, cluster_y, cluster_tof)
 
     def onEvent(self, event):
+        print('IN ONEVENT')
```
Comment (BenMoon) [inline pymepixviewer/panels/blobview.py]: This should probably be `logging` if the output is needed.

**Review by BenMoon — CHANGES_REQUESTED** [review]

File: pymepixviewer/main.py (hunk 1 above)
Comment (samartse) [reply pymepixviewer/main.py]: It is outdated.

File: pymepixviewer/main.py (hunk 2 above)
Comment (samartse) [reply pymepixviewer/main.py]: The same, outdated.

File: pymepixviewer/panels/blobview.py (hunk 3 above)
Comment (samartse) [reply pymepixviewer/panels/blobview.py]: Outdated.

Comment (BenMoon) [reply pymepixviewer/panels/blobview.py]: Sorry, I didn't realize, I was looking at code from an old commit.

**Pull request merged**